datasets/peer_read/peer_read.py

```diff
@@ -0,0 +1,306 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A Dataset of Peer Reviews (PeerRead): Collection, Insights and NLP Applications"""
+
+from __future__ import absolute_import, division, print_function
+
+import glob
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{kang18naacl,
+  title = {A Dataset of Peer Reviews (PeerRead): Collection, Insights and NLP Applications},
+  author = {Dongyeop Kang and Waleed Ammar and Bhavana Dalvi and Madeleine van Zuylen and Sebastian Kohlmeier and Eduard Hovy and Roy Schwartz},
+  booktitle = {Meeting of the North American Chapter of the Association for Computational Linguistics (NAACL)},
+  address = {New Orleans, USA},
+  month = {June},
+  url = {https://arxiv.org/abs/1804.09635},
+  year = {2018}
+}
+"""
+
+_DESCRIPTION = """\
+PearRead is a dataset of scientific peer reviews available to help researchers study this important artifact. The dataset consists of over 14K paper drafts and the corresponding accept/reject decisions in top-tier venues including ACL, NIPS and ICLR, as well as over 10K textual peer reviews written by experts for a subset of the papers.
+"""
+
+_HOMEPAGE = "https://github.com/allenai/PeerRead"
+
+_LICENSE = "Creative Commons Public License"
+
+_URLs = {
+    "dataset_repo": "https://github.com/allenai/PeerRead/archive/master.zip",
+}
+
+
+class PeerRead(datasets.GeneratorBasedBuilder):
+    """A Dataset of Peer Reviews (PeerRead): Collection, Insights and NLP Applications"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="parsed_pdfs",
+            version=VERSION,
+            description="This part of my dataset covers a first domain",
+        ),
+        datasets.BuilderConfig(
+            name="reviews",
+            version=VERSION,
+            description="This part of my dataset covers a second domain",
+        ),
```

Comment:
can you add descriptions of each config here ?

Reply:
Ah sorry I missed this! Done :)